provider/do/build.go

```diff
@@ -51,25 +41,3 @@ func (p *Provider) BuildLogs(app, id string, opts structs.LogsOptions) (io.ReadC
 		}
 	}
 }
-
-func (p *Provider) authAppRepository(app string) error {
-	repo, _, err := p.RepositoryHost(app)
-	if err != nil {
-		return err
-	}
-
-	user, pass, err := p.RepositoryAuth(app)
-	if err != nil {
-		return err
-	}
-
-	cmd := exec.Command("docker", "login", "-u", user, "--password-stdin", repo)
-
-	cmd.Stdin = strings.NewReader(pass)
-
-	if err := cmd.Run(); err != nil {
-		return err
-	}
-
-	return nil
-}
```

Comment:
Is it used to log into 3rd party repositories? Is it going to be needed with BuildKit?

Reply:
:thinking: that's a good point. I'll do some tests

Reply:
it's already being handled here https://github.com/convox/convox/blob/419d89bc28a8375bdab6eb725cb49d063376234b/provider/k8s/build.go#L193